### DISCUSSION TITLE
Group By in cottontaildb

### DISCUSSION THREAD

**Discussion by lutharsanen** [vitrivr/cottontaildb#80] [Q&A]
Dear devs,

We are looking for a way to do a group by in the cottontaildb. Is there a way to do a groupby clause and then count the rows like in a relational Database?

Thank you for your reply!

**Comment by ppanopticon** [comment]
I'm afraid there is currently no GROUP BY in the SQL sense. So far, Cottontail DB only supports simple aggregations over queries.
